hooks/pre-commit

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+{
```

Comment:
Option to possibly provide a nicer error message to someone who doesn't see this in the README.
```bash
{ which git-secrets; } || {
  echo 'git-secrets required and not found. See README for installation instructions.'
  echo
  exit 1
}
```